java-runtime/devel.go

```diff
@@ -0,0 +1,12 @@
+//go:build !release
+
+package kotlinruntime
+
+import (
+	"archive/zip"
+
+	"github.com/TBD54566975/ftl/internal"
+)
+
+// Files is the FTL Java runtime scaffolding files.
+func Files() *zip.Reader { return internal.ZipRelativeToCaller("scaffolding") }
```

Comment:
I don't think this directory exists does it? Can we get rid of Files() altogether?

Reply:
I will remove it, I can re-add when it I add scaffolding.